### DISCUSSION TITLE
Help Radio player for .m3u and .lts file

### DISCUSSION THREAD

**Discussion by abdulsamie10** [community/community#129717] [Programming Help]
### Select Topic Area

Question

### Body

I want help regarding Android Native Radio player for .m3u and .lts file. Its running perfectly for .m3u file but not for .m3u and .lts files.

**Comment by ebndev** [comment]
Hi @abdulsamie10, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share code snippets or a link to your repo. Giving a few more details might help someone give you a nudge in the right direction. 😄
